browser/ui/views/frame/brave_browser_view_layout.cc

```diff
@@ -204,14 +207,19 @@ void BraveBrowserViewLayout::LayoutInfoBar(gfx::Rect& available_bounds) {
 }
 
 void BraveBrowserViewLayout::LayoutContentsContainerView(
-    int top,
-    int bottom,
-    const gfx::Rect& browser_view_bounds) {
-  auto new_rect = browser_view_bounds;
+    const gfx::Rect& available_bounds) {
+  gfx::Rect new_rect = available_bounds;
   if (vertical_tab_strip_host_) {
+    // Both vertical tab impls should not be enabled together.
+    CHECK(!tabs::AreVerticalTabsEnabled());
```

Comment:
I'm confused, it `vertical_tab_strip_host_` is not null then vertical tabs are not enabled? Is this an upstream implementation that came after ours and if so, do we have an open issue to use theirs instead?

Reply:
cc @simonhong @sangwoo108

Reply:
Filed https://github.com/brave/brave-browser/issues/48373 and added it as a comment.
`vertical_tab_strip_host_` is our vertical tab impls. We should not enable ours and chromium's vertical tab impls both. Need to unify both.